src/NUnitFramework/framework/Assert.Conditions.cs

```diff
@@ -32,32 +34,29 @@ public abstract partial class Assert
         #region True
 
         /// <summary>
-        /// Asserts that a condition is true. If the condition is false the method throws
-        /// an <see cref="AssertionException"/>.
+        /// Asserts that a condition is true. Returns without throwing an exception when inside a multiple assert block.
```

Comment:
Folks are going to ask why we don't add `[DoesNotReturnIf(...)]` and other postconditions unless we document that no postconditions are possible due to the multiple assert feature.